app/src/main/res/values-ja/strings.xml

```diff
@@ -0,0 +1,146 @@
+    <string name="snack_engage_c3nav_action">今すぐ入手！</string>
+    <string name="trace_droid_dialog_title">"         <xliff:g example="33C3 Schedule" id="appName">%s</xliff:g>にクラッシュレポート &#8230;を送信します。     "</string>
+    <string name="trace_droid_dialog_message">"         アプリの問題が検出されました。 問題の原因を特定できるように、クラッシュレポートを送信頂ければ幸いです。 クラッシュがいつどのように発生したかを説明できれば、非常に役立ちます。"</string>
+    <string name="trace_droid_button_title_send">メールを開く</string>
+    <string name="trace_droid_no_email_app">"         互換性のあるメールアプリがインストールされていません。"</string>
+    <string name="trace_droid_button_title_no">いいえ</string>
+    <string name="trace_droid_button_title_later">多分後で</string>
+    <string name="notifications_session_alarm_channel_description">さまざまなイベントに対してユーザーが作成したアラーム。</string>
+    <string name="notifications_session_alarm_channel_name">イベントのアラーム</string>
+    <string name="notifications_session_alarm_content_text">イベントのアラーム</string>
+    <string name="notifications_schedule_update_channel_name">スケジュールの更新</string>
+    <string name="alarm_time_title_at_start_time">開始時</string>
+    <string name="alarm_time_title_5_minutes_before">5分前</string>
+    <string name="alarm_time_title_10_minutes_before">10分前</string>
+    <string name="alarm_time_title_15_minutes_before">15分前</string>
+    <string name="alarm_time_title_20_minutes_before">20分前</string>
+    <string name="alarm_time_title_30_minutes_before">30分前</string>
+    <string name="alarm_time_title_45_minutes_before">45分前</string>
+    <string name="alarm_time_title_60_minutes_before">60分前</string>
+    <string name="session_list_item_language_portuguese_content_description">ポルトガル語</string>
+    <string name="session_list_item_language_german_content_description">ドイツ語</string>
+    <string name="session_list_item_language_english_content_description">英語</string>
+    <string name="session_list_item_language_unknown_content_description">未知の言語</string>
+    <string name="session_list_item_language_undefined_content_description">言語: <xliff:g example="es" id="language_abbreviation">%s</xliff:g></string>
+    <string name="session_list_item_language_removed_content_description">言語情報が削除されました</string>
+    <string name="session_list_item_no_video_content_description">ビデオ録画なし</string>
+    <string name="session_list_item_video_content_description">ビデオ録画あり</string>
+    <string name="session_list_item_without_video_content_description">ビデオ録画なし</string>
+    <string name="validation_error_invalid_url">"         無効の <xliff:g example="Engelsystem" id="urlType">%s</xliff:g> URL.     "</string>
+    <string name="validation_error_url_without_query">"         <xliff:g example="Engelsystem" id="urlType">%s</xliff:g> クエリ部分のないURL。 "</string>
+    <string name="validation_error_url_with_incomplete_query">"         <xliff:g example="Engelsystem" id="urlType">%s</xliff:g> クエリ部分が不完全なURL。"</string>
+    <string name="validation_error_url_without_api_key">"         <xliff:g example="Engelsystem" id="urlType">%s</xliff:g> APIキーのないURL。"</string>
```

Comment:
Please remove the surrounding quotes and whitespace. It might have been added by the editor.

Reply:
Thanks for removing the quotes. The whitespace can be removed, too.